tests/main.nf.test

```diff
@@ -20,29 +19,56 @@ nextflow_pipeline {
             assert path("$launchDir/results").exists()
 
             // check FASTP_TRIM JSON outputs from paired and single reads
-            def fastp_JSON_paired = path("$launchDir/results/fastp/SAMPLE1.json").json
+            def fastp_JSON_paired = path("$launchDir/results/fastp/SAMPLE1.fastp.json").json
 
             assert fastp_JSON_paired.summary.sequencing == "paired end (126 cycles + 126 cycles)"
             assert fastp_JSON_paired.filtering_result.passed_filter_reads == 99594
 
-            def fastp_JSON_single = path("$launchDir/results/fastp/SAMPLE3.json").json
+            def fastp_JSON_single = path("$launchDir/results/fastp/SAMPLE3.fastp.json").json
 
             assert fastp_JSON_single.summary.sequencing == "single end (126 cycles)"
             assert fastp_JSON_single.filtering_result.passed_filter_reads == 49986
 
-            // check ADJUST_BRACKEN outputs
+            // check output files
             def lines = []
 
+            // KRAKEN2 outputs
+            lines = path("$launchDir/results/kraken2/SAMPLE2_kraken2_report.txt").text
+
+            assert lines.contains("97.38	24334	24334	U	0	unclassified")
+            assert lines.contains(" 2.53	631	0	S	562	                Escherichia coli")
+            assert lines.contains(" 2.53	631	631	S2	511145	                    Escherichia coli str. K-12 substr. MG1655")
+            assert lines.contains(" 0.09	22	0	S	817	                    Bacteroides fragilis")
+            assert lines.contains(" 0.09	22	22	S1	295405	                      Bacteroides fragilis YCH46")
+
+            // BRACKEN outputs
+            lines = path("$launchDir/results/bracken/SAMPLE2_S_bracken.txt").text
+
+            assert lines.contains("96.63	631	631	S	562	                Escherichia coli")
+            assert lines.contains("3.37	22	22	S	817	                    Bacteroides fragilis")
+
+            lines = path("$launchDir/results/bracken/SAMPLE2_S_bracken_abundances_unsorted.tsv").text
+
+            assert lines.contains("Escherichia coli	562	S	631	0	631	0.96631")
+            assert lines.contains("Bacteroides fragilis	817	S	22	0	22	0.03369")
+
+            // ADJUST_BRACKEN outputs
             lines = path("$launchDir/results/adjust/SAMPLE2_S_bracken_abundances.csv").readLines()
 
-            assert lines.contains("SAMPLE2,unclassified,0,U,24334,0,24334,97.38664")
-            assert lines.contains("SAMPLE2,Escherichia coli,562,S,631,0,631,2.52531")
-            assert lines.contains("SAMPLE2,Bacteroides fragilis,817,S,22,0,22,0.08805")
+            assert lines.contains("SAMPLE2,unclassified,0,U,24334,0,24334,97.37885")
+            assert lines.contains("SAMPLE2,Escherichia coli,562,S,631,0,631,2.52511")
```

Comment:
In the Bracken `abundances_unsorted.tsv`, the final column is given as a proportion (i.e., `0.96631`). But, in this file the final column is given as a percentage (`2.52511`).

Could you modify the adjust bracken abundances step to report proportions instead of percentages so it matches?

This would match the behaviour of the existing pipeline in IRIDA Next (which reports proportions instead of percentages).

Reply:
Done and done 😉 : [5071d42](https://github.com/phac-nml/speciesabundance/pull/13/commits/5071d42aaf8e0a1c79ac804e8311afa038de78e7)